docs/admin_api/user_admin_api.rst

```diff
@@ -1,3 +1,46 @@
+List Accounts
+=============
+
+This API returns all local user accounts.
+
+The api is::
+
+    GET /_synapse/admin/v2/users?offset=0&limit=10&guests=false
+
+including an ``access_token`` of a server admin.
+The parameters ``offset`` and ``limit`` are required only for pagination.
+Per default a ``limit`` of 100 is used. If the endpoint returns less entries
+than specified by ``limit`` then there are no more users left.
+The parameter ``name`` can be used to filter by user name.
```

Comment:
can you document the format of the parameter? is it a regular expression or does it have to match exactly?

Reply:
@richvdh It is used in a `WHERE ? LIKE ?` expression in the SQL query. So `%` and `_` are valid regexp patterns. Do we need to take care of SQL injection for the filters or is it enough to use parse_string(request, ...)?

Reply:
> It is used in a WHERE ? LIKE ? expression in the SQL query. So % and _ are valid regexp patterns

yup, can you add this to the documentation please

> Do we need to take care of SQL injection for the filters or is it enough to use parse_string(request, ...)?

I believe that as long as you use `?` parameters for the sql rather than inlining the parameters into the sql, you are safe from SQL injection.